src/v2/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx

```diff
@@ -333,6 +348,12 @@ const artworkFilterReducer = (
         }
       }
 
+      arrayFilterTypes.forEach((filter) => {
+        if (name === filter) {
+          filterState[name as any] = []
```

Comment:
Can this `name` be `as unknown` or `as `string`?

Reply:
sure. i just copied the one from below.